.github/container/get-source.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+## Fetch a Python package from a git repo and write the pip-tools input manifest to stdout
+## Example:
+## get-source.sh -f https://github.com/google/flax.git -r main -d /opt/flax
+## Output:
+## -e /opt/flax
+
+## Parse command-line arguments
+
+usage() {
+    echo "Usage: $0 [OPTION]..."
+    echo "  -d, --dir PATH         [Required] Local path to check out the source code."
+    echo "  -f, --from URL         [Required] URL of the source repo."
+    echo "  -h, --help             Print usage."
+    echo "  -i, --install          Install the package immediately using pip install."
```

Comment:
Looks like --install and --manifest are mutually exclusive; was this intended? Under what scenario would we use --install?